features/events/daemon/src/main/java/org/opennms/netmgt/eventd/StandardExpandableParameterResolvers.java

```diff
@@ -238,6 +238,28 @@ public String getValue(String parm, String parsedParm, Event event, EventUtil ev
         }
     },
 
+    PRIMARY_INTERFACE {
+
+        @Override
+        public boolean matches(final String parm) {
+            return AbstractEventUtil.TAG_PRIMARY_INTERFACE_ADDRESS.equals(parm);
+        }
+
+        @Override
+        public String getValue(final String parm, final String parsedParm, final Event event, final EventUtil eventUtil) {
+            if (event.getNodeid() != null) {
+                try {
+                    return eventUtil.getPrimaryInterface(event.getNodeid());
+                } catch (SQLException ex) {
+                    // do nothing
+                    LoggerFactory.getLogger(getClass()).info("primary interface ipaddr unavailable for node with id:", event.getNodeid(), ex);
```

Comment:
```suggestion
                    LOG.info("primary interface ipaddr unavailable for node with id: {}", event.getNodeid(), ex);
```

Missing `{}` placeholder.

Would also be a good opportunity to refactor the existing `LoggerFactory.getLogger(getClass())` statements to use a static logger instead.